content/spire/try/oidc_federation_aws.md

```diff
@@ -0,0 +1,399 @@
+
+Complete installation and configuration of the SPIRE Server and Agent are not described in this document. See the [SPIRE documentation](/spire/docs/) for installation and configuration information.
+
+If you are new to SPIFFE and SPIRE, see the [SPIFFE overview](/spiffe/) and [SPIRE introductory documents](/spire/).
+
+## Required Accounts and Permissions
+
+To configure OIDC federation for access to AWS, you'll need the following:
+
+* [SPIRE Server and Agent](/downloads/) executables
+* Ability to create instances to contain the SPIRE Server and Agent
+* Root access to the instances that contain the SPIRE Server and Agent
+* Network access to an ACME-compliant CA such as Let's Encrypt
+* Access to the AWS console
+* [AWS CLI tools](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) installed on the same instance as the SPIRE Agent
+* DNS A record and TCP/IP access to port 443 for the SPIRE OIDC [Discovery Provider REST API](/spire/docs/oidc_discovery/)
+
+# Part One: Configure SPIRE Components
+
+In the first half of this procedure, you configure the SPIRE components. In the second half, you configure the AWS components.
+
+## Configuration Overview for SPIRE Components
+
+The table below summarizes the steps needed to configure the SPIRE components of OIDC federation to AWS. Follow the links for details.
+
+| Step | Action | Description |
+| --- | --- | --- |
+| Step 1 | [Prepare installation environments](#prepare-installation-environments) | Decide where you'll install SPIRE components |
+| Step 2 | [Install, configure, and run the SPIRE Server](#install-configure-and-run-the-spire-server) | Customize `server.conf`, including `ca_key_type` and `jwt_issuer` settings |
+| Step 3 | [Install, configure, and run the OIDC Discovery Provider](#install-configure-and-run-the-oidc-discovery-provider) | The SPIRE OIDC Discovery Provider serves OIDC discovery documents via a REST API |
+| Step 4 | [Install and configure SPIRE Agent](#install-and-configure-the-spire-agent) | Configure the SPIRE Agent as needed for your environment |
+| Step 5 | [Register (Attest) the Node](#register-attest-the-node) | Verify the identity of the node (machine) that the SPIRE Agent runs on |
+| Step 6 | [Register Workloads on the SPIRE Server](#register-workloads-on-the-spire-server) | Verify the identity of workloads |
+
+# Configure SPIRE for OIDC Federation
+
+Follow the procedures in the sections below to configure SPIRE for OIDC federation.
+
+## Prepare Installation Environments
+
+In a typical deployment, you would configure one machine to run the SPIRE Server and OIDC Discovery Provider and a second machine to run the SPIRE Agent. The machines can be a physical machine, virtual machine, instance, or container. The examples in these instructions do not describe how to use Kubernetes or Docker commands to set up OIDC federation, but they are supported.
```

Comment:
Rather than describe here all the ways you _can_ deploy this tutorial, we should simply pick a particular path (eg. deploying to Kubernetes, or running everything on a single Linux VM) and make it clear that _this_ tutorial is in that environment.

The paragraph above might work well in the ODIC Discovery Provider however.